### ISSUE TITLE
Context (parameter) keyword causes unexpected formatting of the function parameters

### ISSUE THREAD

**Issue by eygraber** [pinterest/ktlint#3120]
This is the context parameter equivalent of https://github.com/pinterest/ktlint/issues/2854

## Expected Behavior

All following formattings should be accepted:

    fun test(block: context(param: Int) () -> Unit = {}) {}

    fun test(
        block: context(param: Int) () -> Unit = {},
    ) {}

## Observed Behavior

ktlint formats both cases as follows:

    fun test(
        block: context(param: Int)
        () -> Unit = {},
    ) {}

## Your Environment
* Version of ktlint used: 1.7.1
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): command line

**Comment by paul-dingemans** [comment]
I cannot reproduce the problem. What is your `.editorconfig`?

**Comment by eygraber** [comment]
```
# noinspection EditorConfigKeyCorrectness
[*.{kt,kts}]
ij_kotlin_allow_trailing_comma = true
ij_kotlin_allow_trailing_comma_on_call_site = true
ij_kotlin_imports_layout=*,java.**,javax.**,kotlin.**,^
indent_size = 2
indent_style = space
insert_final_newline = true
ktlint_code_style = android_studio
ktlint_experimental = enabled
ktlint_ignore_back_ticked_identifier = true
ktlint_standard_annotation = disabled
ktlint_standard_blank-line-between-when-conditions = disabled
ktlint_standard_class-signature = disabled
ktlint_standard_comment-wrapping = disabled
# disabled because of https://github.com/pinterest/ktlint/issues/2182#issuecomment-1863408507
ktlint_standard_condition-wrapping = disabled
ktlint_standard_filename = disabled
ktlint_standard_function-naming = disabled
ktlint_standard_function-signature = disabled
ktlint_standard_keyword-spacing = disabled
ktlint_standard_package-name = disabled
ktlint_standard_property-naming = disabled
ktlint_standard_spacing-between-declarations-with-annotations = disabled
max_line_length = 120
```

**Comment by paul-dingemans** [comment]
Still cannot reproduce the problem even when I use your `.editorconfig` setttings. This is the full debug logging that I get when formatting your code example with Ktlint CLI 1.7.1:
```
$ ktlint-1.7.1 --relative --log-level=debug **/Foo.kt
17:53:49.781 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered RuleSetProviderV3 with id 'standard' in ktlint JAR
17:53:49.873 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered RuleSetProviderV3 with id 'standard' in ktlint JAR
17:53:49.877 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'baseline' in ktlint JAR
17:53:49.877 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'plain' in ktlint JAR
17:53:49.877 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'checkstyle' in ktlint JAR
17:53:49.877 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'json' in ktlint JAR
17:53:49.877 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'format' in ktlint JAR
17:53:49.877 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'html' in ktlint JAR
17:53:49.877 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'plain-summary' in ktlint JAR
17:53:49.877 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'sarif' in ktlint JAR
17:53:49.878 [main] DEBUG com.pinterest.ktlint.cli.internal.ReporterAggregator -- Initializing "plain" reporter with {plain=true, color=false, color_name=DARK_GRAY, format=false}
17:53:49.879 [main] DEBUG com.pinterest.ktlint.cli.internal.FileUtils -- Start walkFileTree from directory: '/Users/pdingemans/git/ktlint/ktlint-examples'
17:53:49.886 [main] DEBUG com.pinterest.ktlint.cli.internal.FileUtils -- Discovered 1 files to be processed in 6 ms
17:53:49.892 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.CodeFormatter -- Starting with processing file 'Foo.kt'
17:53:50.048 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.EditorConfigLoader -- Effective editorconfig properties for file '/Users/pdingemans/git/ktlint/ktlint-examples/src/main/kotlin/Foo.kt':
        ij_kotlin_allow_trailing_comma: true
        ij_kotlin_allow_trailing_comma_on_call_site: true
        ij_kotlin_imports_layout: *,java.**,javax.**,kotlin.**,^
        indent_size: 2
        indent_style: space
        insert_final_newline: true
        ktlint_code_style: android_studio
        ktlint_experimental: enabled
        ktlint_ignore_back_ticked_identifier: true
        ktlint_standard_annotation: disabled
        ktlint_standard_blank-line-between-when-conditions: disabled
        ktlint_standard_class-signature: disabled
        ktlint_standard_comment-wrapping: disabled
        ktlint_standard_condition-wrapping: disabled
        ktlint_standard_filename: disabled
        ktlint_standard_function-naming: disabled
        ktlint_standard_function-signature: disabled
        ktlint_standard_keyword-spacing: disabled
        ktlint_standard_package-name: disabled
        ktlint_standard_property-naming: disabled
        ktlint_standard_spacing-between-declarations-with-annotations: disabled
        max_line_length: 120
        tab_width: 2
17:53:50.052 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:argument-list-wrapping' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.052 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:argument-list-wrapping' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:function-literal' should run after the rule with id 'standard:chain-method-continuation'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:modifier-list-spacing' should run after the rule with id 'standard:annotation'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:wrapping' should run after the rule with id 'standard:annotation'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:argument-list-wrapping' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:argument-list-wrapping' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.053 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
17:53:50.056 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.RuleProviderSorter -- Rules will be executed in order below:
           - internal:ktlint-suppression, 
           - standard:annotation-spacing, 
           - standard:backing-property-naming, 
           - standard:binary-expression-wrapping, 
           - standard:chain-wrapping, 
           - standard:class-naming, 
           - standard:colon-spacing, 
           - standard:comma-spacing, 
           - standard:comment-spacing, 
           - standard:context-receiver-list-wrapping, 
           - standard:context-receiver-wrapping, 
           - standard:curly-spacing, 
           - standard:discouraged-comment-location, 
           - standard:dot-spacing, 
           - standard:double-colon-spacing, 
           - standard:enum-entry-name-case, 
           - standard:enum-wrapping, 
           - standard:expression-operand-wrapping, 
           - standard:final-newline, 
           - standard:fun-keyword-spacing, 
           - standard:function-expression-body, 
           - standard:function-return-type-spacing, 
           - standard:function-start-of-body-spacing, 
           - standard:function-type-modifier-spacing, 
           - standard:function-type-reference-spacing, 
           - standard:import-ordering, 
           - standard:kdoc, 
           - standard:kdoc-wrapping, 
           - standard:modifier-order, 
           - standard:multiline-if-else, 
           - standard:multiline-loop, 
           - standard:no-blank-line-before-rbrace, 
           - standard:no-blank-lines-in-chained-method-calls, 
           - standard:no-consecutive-blank-lines, 
           - standard:no-empty-class-body, 
           - standard:no-empty-file, 
           - standard:no-empty-first-line-in-method-block, 
           - standard:no-line-break-after-else, 
           - standard:no-line-break-before-assignment, 
           - standard:no-multi-spaces, 
           - standard:no-trailing-spaces, 
           - standard:no-unit-return, 
           - standard:no-wildcard-imports, 
           - standard:nullable-type-spacing, 
           - standard:op-spacing, 
           - standard:parameter-list-spacing, 
           - standard:parameter-list-wrapping, 
           - standard:parameter-wrapping, 
           - standard:paren-spacing, 
           - standard:property-wrapping, 
           - standard:range-spacing, 
           - standard:spacing-around-angle-brackets, 
           - standard:spacing-between-declarations-with-comments, 
           - standard:spacing-between-function-name-and-opening-parenthesis, 
           - standard:square-brackets-spacing, 
           - standard:statement-wrapping, 
           - standard:string-template, 
           - standard:type-argument-comment, 
           - standard:type-argument-list-spacing, 
           - standard:type-parameter-comment, 
           - standard:type-parameter-list-spacing, 
           - standard:unary-op-spacing, 
           - standard:unnecessary-parentheses-before-trailing-lambda, 
           - standard:value-argument-comment, 
           - standard:value-parameter-comment, 
           - standard:function-literal, 
           - standard:modifier-list-spacing, 
           - standard:wrapping, 
           - standard:argument-list-wrapping, 
           - standard:no-semi, 
           - standard:trailing-comma-on-call-site, 
           - standard:trailing-comma-on-declaration-site, 
           - standard:indent, 
           - standard:block-comment-initial-star-alignment, 
           - standard:max-line-length
17:53:50.073 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.CodeFormatter -- Finished with processing file 'Foo.kt'
src/main/kotlin/Foo.kt:4:1: Unexpected indentation (4) (should be 2) (standard:indent)
17:53:50.073 [main] WARN com.pinterest.ktlint.cli.internal.KtlintCommandLine -- Lint has found errors than can be autocorrected using 'ktlint --format'

Summary error count (descending) by rule:
  standard:indent: 1
17:53:50.074 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintCommandLine -- Finished processing in 296ms / 1 file(s) scanned / 1 error(s) found
17:53:50.074 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintCommandLine -- Exit ktlint with exit code: 1
```

Please send me your output, if you can still reproduce.

**Comment by eygraber** [comment]
I can still repro, and here is the output:

```
12:06:11.238 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered RuleSetProviderV3 with id 'standard' in ktlint JAR
12:06:11.349 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered RuleSetProviderV3 with id 'standard' in ktlint JAR
12:06:11.355 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'baseline' in ktlint JAR
12:06:11.356 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'plain' in ktlint JAR
12:06:11.356 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'checkstyle' in ktlint JAR
12:06:11.356 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'json' in ktlint JAR
12:06:11.356 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'format' in ktlint JAR
12:06:11.356 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'html' in ktlint JAR
12:06:11.356 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'plain-summary' in ktlint JAR
12:06:11.356 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintServiceLoader -- Discovered ReporterProviderV2 with id 'sarif' in ktlint JAR
12:06:11.356 [main] DEBUG com.pinterest.ktlint.cli.internal.ReporterAggregator -- Initializing "plain" reporter with {plain=true, color=false, color_name=DARK_GRAY, format=false}
12:06:11.369 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.CodeFormatter -- Starting with processing file 'CaseTasksNavigator.kt'
12:06:11.585 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.EditorConfigLoader -- Effective editorconfig properties for file '/home/eli/workspace/case-status-android3/screens/case-tasks/src/main/kotlin/com/myapp/screens/cases/tasks/CaseTasksNavigator.kt':
        ij_kotlin_allow_trailing_comma: true
        ij_kotlin_allow_trailing_comma_on_call_site: true
        ij_kotlin_imports_layout: *,java.**,javax.**,kotlin.**,^
        indent_size: 2
        indent_style: space
        insert_final_newline: true
        ktlint_code_style: android_studio
        ktlint_experimental: enabled
        ktlint_ignore_back_ticked_identifier: true
        ktlint_standard_annotation: disabled
        ktlint_standard_blank-line-between-when-conditions: disabled
        ktlint_standard_class-signature: disabled
        ktlint_standard_comment-wrapping: disabled
        ktlint_standard_condition-wrapping: disabled
        ktlint_standard_filename: disabled
        ktlint_standard_function-naming: disabled
        ktlint_standard_function-signature: disabled
        ktlint_standard_keyword-spacing: disabled
        ktlint_standard_package-name: disabled
        ktlint_standard_property-naming: disabled
        ktlint_standard_spacing-between-declarations-with-annotations: disabled
        max_line_length: 120
        tab_width: 2
12:06:11.595 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:argument-list-wrapping' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.595 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:argument-list-wrapping' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.595 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:function-literal' should run after the rule with id 'standard:chain-method-continuation'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.595 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.595 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.595 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:modifier-list-spacing' should run after the rule with id 'standard:annotation'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.596 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:wrapping' should run after the rule with id 'standard:annotation'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.596 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:argument-list-wrapping' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.596 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:argument-list-wrapping' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.596 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.596 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.596 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:class-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.596 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter -- Rule with id 'standard:indent' should run after the rule with id 'standard:function-signature'. However, the latter rule is not loaded and is allowed to be ignored. For best results, it is advised load the rule.
12:06:11.603 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.RuleProviderSorter -- Rules will be executed in order below:
           - internal:ktlint-suppression, 
           - standard:annotation-spacing, 
           - standard:backing-property-naming, 
           - standard:binary-expression-wrapping, 
           - standard:chain-wrapping, 
           - standard:class-naming, 
           - standard:colon-spacing, 
           - standard:comma-spacing, 
           - standard:comment-spacing, 
           - standard:context-receiver-list-wrapping, 
           - standard:context-receiver-wrapping, 
           - standard:curly-spacing, 
           - standard:discouraged-comment-location, 
           - standard:dot-spacing, 
           - standard:double-colon-spacing, 
           - standard:enum-entry-name-case, 
           - standard:enum-wrapping, 
           - standard:expression-operand-wrapping, 
           - standard:final-newline, 
           - standard:fun-keyword-spacing, 
           - standard:function-expression-body, 
           - standard:function-return-type-spacing, 
           - standard:function-start-of-body-spacing, 
           - standard:function-type-modifier-spacing, 
           - standard:function-type-reference-spacing, 
           - standard:import-ordering, 
           - standard:kdoc, 
           - standard:kdoc-wrapping, 
           - standard:modifier-order, 
           - standard:multiline-if-else, 
           - standard:multiline-loop, 
           - standard:no-blank-line-before-rbrace, 
           - standard:no-blank-lines-in-chained-method-calls, 
           - standard:no-consecutive-blank-lines, 
           - standard:no-empty-class-body, 
           - standard:no-empty-file, 
           - standard:no-empty-first-line-in-method-block, 
           - standard:no-line-break-after-else, 
           - standard:no-line-break-before-assignment, 
           - standard:no-multi-spaces, 
           - standard:no-trailing-spaces, 
           - standard:no-unit-return, 
           - standard:no-wildcard-imports, 
           - standard:nullable-type-spacing, 
           - standard:op-spacing, 
           - standard:parameter-list-spacing, 
           - standard:parameter-list-wrapping, 
           - standard:parameter-wrapping, 
           - standard:paren-spacing, 
           - standard:property-wrapping, 
           - standard:range-spacing, 
           - standard:spacing-around-angle-brackets, 
           - standard:spacing-between-declarations-with-comments, 
           - standard:spacing-between-function-name-and-opening-parenthesis, 
           - standard:square-brackets-spacing, 
           - standard:statement-wrapping, 
           - standard:string-template, 
           - standard:type-argument-comment, 
           - standard:type-argument-list-spacing, 
           - standard:type-parameter-comment, 
           - standard:type-parameter-list-spacing, 
           - standard:unary-op-spacing, 
           - standard:unnecessary-parentheses-before-trailing-lambda, 
           - standard:value-argument-comment, 
           - standard:value-parameter-comment, 
           - standard:function-literal, 
           - standard:modifier-list-spacing, 
           - standard:wrapping, 
           - standard:argument-list-wrapping, 
           - standard:no-semi, 
           - standard:trailing-comma-on-call-site, 
           - standard:trailing-comma-on-declaration-site, 
           - standard:indent, 
           - standard:block-comment-initial-star-alignment, 
           - standard:max-line-length
12:06:11.644 [pool-1-thread-1] DEBUG com.pinterest.ktlint.rule.engine.internal.CodeFormatter -- Finished with processing file 'CaseTasksNavigator.kt'
/home/eli/workspace/case-status-android3/screens/case-tasks/src/main/kotlin/com/myapp/screens/cases/tasks/CaseTasksNavigator.kt:10:65: Expected a newline after the context receiver (standard:context-receiver-wrapping)
12:06:11.645 [main] WARN com.pinterest.ktlint.cli.internal.KtlintCommandLine -- Lint has found errors than can be autocorrected using 'ktlint --format'

Summary error count (descending) by rule:
  standard:context-receiver-wrapping: 1
12:06:11.647 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintCommandLine -- Finished processing in 413ms / 1 file(s) scanned / 1 error(s) found
12:06:11.647 [main] DEBUG com.pinterest.ktlint.cli.internal.KtlintCommandLine -- Exit ktlint with exit code: 1
```

**Comment by eygraber** [comment]
Apologies, I copied the code sample from the previous issue and didn't copy my own code 🙈 

Here's the code it fails with:

```kotlin
class Foo(
  private val foo: context(Int, Int) (Int) -> Unit,
)
```

**Comment by paul-dingemans** [comment]
Thank you. Now it is reproducible. Interestingly is that it is a problem when using in `class` but not in a `function`:

```
class Foo2(
  foo: context(Int, Int) (Int) -> Unit,
)

fun Foo2(
  foo: context(Int, Int) (Int) -> Unit,
)
```
is reformatted as:
```
class Foo2(
  foo: context(Int, Int)
  (Int) -> Unit,
)

fun Foo2(
  foo: context(Int, Int) (Int) -> Unit,
)
```

Your initial examples both used a `function` instead of a class, and as of that did not reproduce.

**Comment by kyay10** [comment]
I'm getting a similar issue in typealiases:
```kotlin
typealias Validator<IN, OUT> = context(ValidationContext) (IN) -> ValidationResult<OUT>
```
formats to:
```kotlin
typealias Validator<IN, OUT> = context(ValidationContext)
(IN) -> ValidationResult<OUT>
```
I'll file a separate issue with more info like config etc
